draft-ietf-httpbis-rfc6265bis.md

```diff
@@ -962,7 +962,17 @@ Note: The port component of the origins is not considered.
 
 A request is "same-site" if its target's URI's origin is same-site with the
 request's client's "site for cookies" (which is an origin), or if the request
```

Comment:
This sentence is difficult to parse, I think removing the comma immediately after "(which is an origin)" should help.

Reply:
Done, removed comma.